ertviz/views/__init__.py

```diff
@@ -1,12 +1,31 @@
 import dash_html_components as html
 import dash_core_components as dcc
 from .paremeter_view import parameter_view
+from .ensemble_selector_view import ensemble_selector_view
+
+
+def _graph_type_div(parent, name="graph_type", keys=[]):
+    return html.Div(
+        children=[
+            html.Label("Select Graph Type"),
+            dcc.RadioItems(
+                options=[{"label": key, "value": key} for key in keys],
+                value=keys[0],
+                id=parent.uuid("plot-type"),
+            ),
+        ]
+    )
 
 
 def response_view(parent):
     return [
         html.Div(
             [
+                _graph_type_div(
```

Comment:
Once changing this, can we move this `_graph_type_div` under the response selector? ie.  
```python
html.Div(
                    [
                        html.H5("Response"),
                        dcc.Dropdown(id=parent.uuid("response-selector")),
                        _graph_type_div(
                            parent=parent,
                            name="graph_type",
                            keys=["Function plot", "Statistics"],
                        ),
                    ],
```